src/main/java/seedu/address/logic/Messages.java

```diff
@@ -16,6 +16,8 @@ public class Messages {
     public static final String MESSAGE_INVALID_COURSE_MATE_NAME = "The courseMate name provided is not found!";
     public static final String MESSAGE_INVALID_COURSE_MATE_DISPLAYED_INDEX = "The courseMate index provided is invalid";
     public static final String MESSAGE_COURSE_MATES_LISTED_OVERVIEW = "%1$d course mates listed!";
+    public static final String MESSAGE_INVALID_GROUP_NAME = "The group name provided is not found!";
+    public static final String MESSAGE_MEMBERS_DONT_EXIST = "Some of the specified members could not be found.";
```

Comment:
To be solved later along with other messages: make it more informative (which of the members are not found)